### ISSUE TITLE
Change images with product variants

### ISSUE THREAD

**Issue by kennedyrose** [smarterlabs/labs#38]

**Comment by tbaustin** [comment]
Are you just wanting images to change when you select a product variant? Let me know exactly what you want this to do and I'll tackle this one this weekend!

**Comment by kennedyrose** [comment]
Correct, all the product images in the carousel should change to the selected product.

**Comment by tbaustin** [comment]
This one seems to be already working! I think you can close this!
